### PR TITLE
Rebased rdf loader 

### PR DESCRIPTION
**Pull request by JafarAbdi**
### Description

A rebased version of this pr-rdf_loader onto pr-moveit_core

### Checklist
- [ ] **Required by CI**: Code is auto formatted using [clang-format](http://moveit.ros.org/documentation/contributing/code)
- [ ] Extended the tutorials / documentation, if necessary [reference](http://moveit.ros.org/documentation/contributing/)
- [ ] Include a screenshot if changing a GUI
- [ ] Document API changes relevant to the user in the moveit/MIGRATION.md notes
- [ ] Created tests, which fail without this PR [reference](http://docs.ros.org/kinetic/api/moveit_tutorials/html/doc/tests.html)
- [ ] Decide if this should be cherry-picked to other current ROS branches
- [ ] While waiting for someone to review your request, please consider reviewing [another open pull request](https://github.com/ros-planning/moveit/pulls) to support the maintainers

[//]: # "You can expect a response from a maintainer within 7 days. If you haven't heard anything by then, feel free to ping the thread. Thank you!"

### CODE REVIEW

**Comment by henningkayser** [comment]
Great work!

File: moveit_ros/planning/rdf_loader/src/rdf_loader.cpp [hunk 1]
```diff
@@ -49,51 +51,82 @@
 #include <fstream>
 #include <streambuf>
 #include <algorithm>
+#include <chrono>
 
-rdf_loader::RDFLoader::RDFLoader(const std::string& robot_description)
+namespace rdf_loader
+{
+rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_rdf_loader.rdf_loader");
```
Comment (henningkayser) [inline moveit_ros/planning/rdf_loader/src/rdf_loader.cpp]:
```suggestion
static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_rdf_loader.rdf_loader");
```

**Comment by JafarAbdi** [comment]
> I see that you added `COLCON_IGNORE` to the other subfolders. I thought that colcon only considers the top-level `CMakeLists` as a package and ignores the others. What happens if you remove the `COLCON_IGNORE` files but only keep `rdf_loader` enabled in the `CMakeLists`?

I added the `COLCON_IGNORE` to the packages of `moveit_ros` [link](https://github.com/ros-planning/moveit2/pull/127/commits/7d8f6fd638930a8c93fa76ca6c50ab498abe4203), except `moveit_ros_planning` and disabled its sub-pacakges in `CMakeLists.txt` [link](https://github.com/ros-planning/moveit2/pull/127/commits/7fd5cef4c226e20262c3420763322332c7cf3e61)

**Comment by JafarAbdi** [comment]
I'll close this PR because I force-pushed to the original PR

**Pull request closed without merging**